### ISSUE TITLE
Generating frontend code is interrupted.

### ISSUE THREAD

**Issue by hihaluemen** [raidendotai/cofounder#76]
Why is it that every time I generate, there are many parts of the frontend that cannot be fully generated when it comes to generating frontend code (the generation is interrupted...)?

![image](https://github.com/user-attachments/assets/5f70920f-3e19-4c67-baae-2cb4ea8fb68e)

**Comment by BioCruz-sudo** [comment]
yeah sometimes it gets interrupted and I try to reset the server but then all the saved work disappears and there goes 5-10$ of API usage
